core/domain/prod_validators_test.py

```diff
@@ -273,265 +271,6 @@ def test_model_with_invalid_id(self):
-        with datastore_services.mock_datetime_for_datastore(mocked_datetime):
-            self.run_job_and_check_output(
-                expected_output, sort=True, literal_eval=False)
-
-    def test_missing_exploration_model_failure(self):
-        exp_models.ExplorationModel.get_by_id('0').delete(
-            feconf.SYSTEM_COMMITTER_ID, '', [])
-        expected_output = [
-            (
-                u'[u\'failed validation check for exploration_ids field '
-                'check of ClassifierTrainingJobModel\', '
-                '[u"Entity id %s: based on field exploration_ids having value '
-                '0, expected model ExplorationModel with id 0 but it doesn\'t '
-                'exist"]]') % self.model_instance_0.id,
-            u'[u\'fully-validated ClassifierTrainingJobModel\', 1]']
-        self.run_job_and_check_output(
-            expected_output, sort=True, literal_eval=False)
-
-    def test_invalid_exp_version(self):
-        self.model_instance_0.exp_version = 5
-        self.model_instance_0.put()
-        expected_output = [
-            (
-                u'[u\'failed validation check for exp version check '
-                'of ClassifierTrainingJobModel\', [u\'Entity id %s: '
-                'Exploration version 5 in entity is greater than the '
-                'version 1 of exploration corresponding to exp_id 0\']]'
-            ) % self.model_instance_0.id,
-            u'[u\'fully-validated ClassifierTrainingJobModel\', 1]']
-        self.run_job_and_check_output(
-            expected_output, sort=True, literal_eval=False)
-
-    def test_invalid_state_name(self):
-        self.model_instance_0.state_name = 'invalid'
-        self.model_instance_0.put()
-        expected_output = [
-            (
-                u'[u\'failed validation check for state name check '
-                'of ClassifierTrainingJobModel\', [u\'Entity id %s: '
-                'State name invalid in entity is not present in '
-                'states of exploration corresponding to exp_id 0\']]'
-            ) % self.model_instance_0.id,
-            u'[u\'fully-validated ClassifierTrainingJobModel\', 1]']
-        self.run_job_and_check_output(
-            expected_output, sort=True, literal_eval=False)
-
-    def test_model_with_invalid_schema(self):
-        self.model_instance_0.interaction_id = 'invalid'
-        self.model_instance_0.put()
-        expected_output = [
-            (
-                u'[u\'failed validation check for domain object check '
-                'of ClassifierTrainingJobModel\', [u\'Entity id %s: Entity '
-                'fails domain validation with the error Invalid '
-                'interaction id: invalid\']]'
-            ) % self.model_instance_0.id,
-            u'[u\'fully-validated ClassifierTrainingJobModel\', 1]']
-        self.run_job_and_check_output(
-            expected_output, sort=True, literal_eval=False)
-
-
-class TrainingJobExplorationMappingModelValidatorTests(
```

Comment:
We can keep this test class here since we're keeping `TrainingJobExplorationMappingModelValidator` in `prod_validators.py`.